Modules/Filtering/ImageGradient/test/itkVectorGradientMagnitudeImageFilterTest1.cxx

```diff
@@ -39,20 +39,22 @@ itkVectorGradientMagnitudeImageFilterTest1(int argc, char * argv[])
     return EXIT_FAILURE;
   }
 
+  constexpr unsigned int Dimension = 2;
+
   using RGBPixelType = itk::RGBPixel<unsigned short>;
-  using CharImageType = itk::Image<unsigned char, 2>;
-  using RGBImageType = itk::Image<RGBPixelType, 2>;
-  using AdaptorType = itk::RGBToVectorImageAdaptor<RGBImageType>;
-  using FilterType = itk::VectorGradientMagnitudeImageFilter<AdaptorType>;
-  using ReaderType = itk::ImageFileReader<RGBImageType>;
-  using RescaleFilterType = itk::RescaleIntensityImageFilter<FilterType::OutputImageType, CharImageType>;
-  using WriterType = itk::ImageFileWriter<CharImageType>;
+  using CharImageType = itk::Image<unsigned char, Dimension>;
+  using RGBImageType = itk::Image<RGBPixelType, Dimension>;
 
   // Create a reader and filter
+  using ReaderType = itk::ImageFileReader<RGBImageType>;
```

Comment:
I don't know the answer to this question ... is there an ITK style for the placement of these `using` statements?  Should they all be near the top of the scope or should they be delayed until their first use?  I've generally been doing the former, which is the opposite of the change here.

Reply:
The ITK SW Guide (https://itk.org/ItkSoftwareGuide.pdf section C.23.1, p. 368) contains the following passage:
```
Note that constant declarations (e.g. image dimensions, etc.) and type alias declarations (e.g. pixel
and image types, etc.) should be local to where they are used for the sake of readability.
```

Maybe it can be improved. Generally, I'd say that we prefer declaring aliases close to where they are used, except for those ones that are used extensively (e.g. the image types) in a test.